### PR TITLE
[Refactor] Shine

### PR DESCRIPTION
**Pull request by joebell93**
## Description
Shine currently only works assuming all animations and positioned the same way (offset appearance on Tiles) 

This should be applied on the context using `translateX()` positioning.

## Related Issue
https://github.com/sky-uk/toolkit/issues/190

## Motivation and Context
Blocks toolkit-react shine integration in Hero - @coderas 

## How Has This Been Tested?
TBC

## Screenshots (if appropriate)
No visual changes

## Types of Changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Browser Support
- [x] Chrome
- [x] Edge
- [x] Firefox
- [x] IE9
- [x] IE10
- [x] IE11
- [x] Opera
- [x] Safari
- [x] Mobile Devices
- [x] Screen-readers (e.g. JAWS, Apple VoiceOver)

## Checklist
- [x] All code includes full inline documentation (as per the [template](https://github.com/sky-uk/toolkit-core/blob/master/_template.scss)).
- [x] All code conforms to the Toolkit [coding style](https://github.com/sky-uk/toolkit/wiki/Coding-Style).
- [x] All code conforms to [WCAG 2.0 level AA Accessibility Guidelines](https://www.w3.org/TR/WCAG20/).
- [x] New functions and mixins have appropriate tests.
- [x] All new and existing tests passed.
- [x] I have added instructions on how to test my changes.
- [x] Package version updated.
- [x] CHANGELOG.md updated.

### CODE REVIEW

File: components/_shine.scss [hunk 1]
```diff
@@ -66,43 +61,49 @@ $shine-offset: 25%;
...
-      left: -$shine-offset;
-    }
+/**
+ * Shine Context
+ *
+ * Applied to the parent node in which the shine is contained, animating shine
+ * rails on hover.
+ */
+.c-shine-context:hover {
+  /*! autoprefixer: off */
```
Comment (aaronthomas) [inline components/_shine.scss]: Is it a little heavy-handed to disable autoprefixer at a toolkit level like this? What if a consumer wants to add `-moz-` prefixes in their build?

For now I think this PR is fine as long as we know we 100% don't need any more prefixes than `-ms-`, but it's worth bearing this in mind moving forward and perhaps trying to find a cleaner solution if we need to do this more often.

**Comment by coderas** [comment]
like this 👍

File: components/_shine.scss (hunk 1 above)
Comment (joebell93) [inline components/_shine.scss]: It's a tricky one, but personally don't feel like people should be able to add additional prefixes - it should just work out of the box (whatever the consumer's building it with)

Comment (joebell93) [inline components/_shine.scss]: Ideally we should support the [`last 3 versions`](http://browserl.ist/?q=last+3+versions)

**Pull request merged**